data/daftar-isi.js

```diff
@@ -8,8 +8,8 @@ export default [
         api: '/api/pancasila',
       },
       {
-        text: 'UUD 1945',
-        href: '/uud1945/',
+        text: 'Pembukaan UUD 1945',
+        href: '/pembukaan-uud-1945/',
```

Comment:
Can you also help to find in all directory for word `/uud1945`, I guess there are other places that using this slug.

Reply:
pushed, kindly check again